docs/.vitepress/Card.vue

```diff
@@ -11,29 +11,17 @@
 // </cards>
 -->
 
-<script setup>
-const props = defineProps({
-  imgurl: String,
-  imgalt: String,
-});
-</script>
-
 <template>
-  <article class="card-wrapper rounded-4 shadow-s overflow-hidden">
-    <div v-if="imgurl" class="flex items-center justify-center gap-8 overflow-hidden img-bg" style="aspect-ratio: 4 / 3">
-      <img :src="imgurl" :alt="imgalt">
-    </div>
-    <div class="do-caption p-16">
-      <slot></slot>
-    </div>
+  <article class="docs-card rounded-4 shadow-s focus-within:shadow-m hover:shadow-m overflow-hidden relative">
+    <slot></slot>
   </article>
 </template>
 
-<style lang="scss">
-.card-wrapper {
-  .img-bg {
-    background-color: #F9F9FA;
-  }
+<style scoped lang="scss">
+
+.docs-card:focus-within{
+  //border: 1px solid red;
```

Comment:
Do we want to keep it if it's commented out? I see there's no navigation feedback now, but if you restore it, the card changes width on focus:
![card-focus](https://github.com/user-attachments/assets/831306ed-101f-4062-b4d7-47b23c343cd6)

Reply:
The staes are just guesswork for now, will go over it when that stuff is ready. There is actually navigation feedback as a shadow thing, hard to see in dark mode, but dark mode is not on the supportplan right now anyways :)